codes/dart/utils/print_util.dart

```diff
@@ -0,0 +1,75 @@
+/**
+ * File: PrintUtil
+ * Created Time: 2023-01-23
+ * Author: Jefferson (JeffersonHuang77@gmail.com)
+ */
+import 'dart:io';
+
+import 'list_node.dart';
+import 'tree_node.dart';
+
+class Trunk {
+  Trunk? prev;
+  String str;
+
+  Trunk(this.prev, this.str);
+}
+
+class PrintUtil {
```

Comment:
Remove the class if it is unnecessary.

(In Java, we have to create a class but it is not elegant.)